addons/website/models/ir_ui_view.py

```diff
@@ -248,7 +248,16 @@ def get_inheriting_views_arch(self, view_id, model):
         if not self._context.get('website_id'):
             return super(View, self).get_inheriting_views_arch(view_id, model)
 
-        inheriting_views = super(View, self.with_context(active_test=False)).get_inheriting_views_arch(view_id, model)
+        get_inheriting_self = self.with_context(active_test=False)
+        if self.pool._init and not self._context.get('load_all_views'):
+            view = self.browse(view_id)
+            if view.website_id:
+                original_view = self.search([('key', '=', view.key), ('website_id', '=', False)])
```

Comment:
This could probably be
```python
original_view = view._get_original_view()
```
Since this is trigered by [bdc76207c#137](https://github.com/odoo/odoo/commit/bdc76207ca2981490a2a1e8c8467f2f7ef01c5e8#diff-bf8d854179fb58216be065c91d78d2b6R137) (which initial diff is https://github.com/odoo/odoo/commit/b5fe23055d808026272fa0806c46cead49612c84) that should be the same?